tests/financial_transcripts/postgres/test_postgres_flow.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+def test_rag_postgres_flow_runs():
+    '''
+    The goal of the test: flow runs without error
```

Comment:
as we discussed over the call, this can be addressed separately. i plan to create a story to explore this feature and apply to this test, this will be helpful as some attendees of our presentations have been asking about the same.